src/components/modals/SubmitTranscriptModal.tsx

```diff
@@ -127,6 +128,19 @@ const SubmitTranscriptModal = ({ submitState, onClose }: Props) => {
                     </Link>
                     .
                   </Text>
+                  <Divider marginY={2} />
+                  <Text fontSize="sm" alignSelf='end'>
+                    Help us improve!{" "}
+                    <Link
+                      href={discordInvites.review_guidelines}
```

Comment:
isn't this supposed to be `discordInvites.feedback`?

Reply:
good catch! updated!